ACM-11562:writing process from upstream new ACM-MCE docs for discover function (#6629)
open-cluster-management/rhacm-docs: clusters/release_notes/whats_new.adoc

[#acm-integration-wn]
== {product-title-short} integration

> * You can now automate the import of {rosa} clusters by using {product-title-short} policy enforcement with {mce-short} for faster cluster management. Learn more at link:../../clusters/acm_integration/acm_integration_import_rosa.adoc#import-discover-rosa[Automating import for discovered {rosa} clusters].

Reviewer: ```suggestion
* You can now automate the import of {rosa} clusters by using {product-title-short} policy enforcement with {mce-short} for faster cluster management. Learn more at link:../../clusters/acm_integration/acm_integrate_import_rosa.adoc#import-discover-rosa[Automating import for discovered {rosa} clusters].
```

> * You can now automate the import of {rosa} clusters by using {product-title-short} policy enforcement with {mce-short} for faster cluster management. Learn more at link:../clusters/acm_integration/acm_integration_import_rosa.adoc#import-discover-rosa[Automating import for discovered {rosa} clusters].
> 
> * You can now discover and automate the import of hosted clusters, as well. Configure  {product-title-short} to import {mce-short} clusters and then use policy enforcement for automation. Learn more at xref:../clusters/acm_integration/acm_discover_hosted.adoc#discover-hosted-acm[Automating import for discovered hosted clusters].

Reviewer: same suggestion about adding `../`
```suggestion
* You can now automate the import of {rosa} clusters by using {product-title-short} policy enforcement with {mce-short} for faster cluster management. Learn more at link:../../clusters/acm_integration/acm_integration_import_rosa.adoc#import-discover-rosa[Automating import for discovered {rosa} clusters].

* You can now discover and automate the import of hosted clusters, as well. Configure  {product-title-short} to import {mce-short} clusters and then use policy enforcement for automation. Learn more at xref:../../clusters/acm_integration/acm_discover_hosted.adoc#discover-hosted-acm[Automating import for discovered hosted clusters].
```

> You can now automate the import of {rosa} clusters by using {product-title-short} policy enforcement with {mce-short} for faster cluster management. Learn more at xref:../clusters/acm_integration/integration_import_rosa.adoc#import-discover-rosa[Automating import for discovered OpenShift Service on AWS clusters].
> 
> You can now discover and automate the import of hosted clusters, as well. Configure  {product-title-short} to import {mce-short} clusters and then use policy enforcement for automation. Learn more at xref:../clusters/acm_integration/acm_discover_hosted.adoc#discovering-hcp-acm[Automating import for discovered hosted clusters].

Reviewer: Broken links here and need to match title
```suggestion
* You can now automate the import of {rosa} clusters by using {product-title-short} policy enforcement with {mce-short} for faster cluster management. Learn more at link:../clusters/acm_integration/acm_integration_import_rosa.adoc#import-discover-rosa[Automating import for discovered {rosa} clusters].

* You can now discover and automate the import of hosted clusters, as well. Configure  {product-title-short} to import {mce-short} clusters and then use policy enforcement for automation. Learn more at xref:../clusters/acm_integration/acm_discover_hosted.adoc#discover-hosted-acm[Automating import for discovered hosted clusters].
```

Author: good catch, I changed them in the main but forgot to go here
